libraries/chain/protocol/custom_authority.cpp

```diff
@@ -59,22 +102,10 @@ void custom_authority_create_operation::validate()const
               "Can not create custom authority for special accounts" );
 
    FC_ASSERT( valid_from < valid_to, "valid_from must be earlier than valid_to" );
-
-   // Note: when adding new operation with hard fork, need to check more strictly in evaluator
```

Comment:
What happend to line 63-71?
I am sure that there was a reason for some these, in particlar the test for `operation_type<operation:count()`

Reply:
Yes, probably operation_type ASSERT should be moved back.
But assert for auth.address_auths was removed because that field was removed.

Reply:
done